RELEASE.md

```diff
@@ -145,6 +145,16 @@ After a new stable `v$MAJOR.$MINOR` release branch has been created, update the
 `target-branch` and `commit-message.prefix` fields in `.github/dependabot.yml`
 to point to the newly created branch, instead of the old stable branch.
 
+## Update the `renovate` configuration
+
+After a new stable `v$MAJOR.$MINOR` release branch has been created, update `renovate.json5`:
```

Comment:
nit:
```suggestion
After a new stable `v$MAJOR.$MINOR` release branch has been created, update `.github/renovate.json5`:
```

Reply:
The original PR put it at `renovate.json` at the root, but your right, I can move it into `.github` I think.